### DISCUSSION TITLE
Content Keywords Condition Invalidation in log collection in skywalking 8.5.0 es7

### DISCUSSION THREAD

**Discussion by IMWalkersun** [apache/skywalking#8546] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/33855218/153976958-63931ef4-0f63-4b63-a9fe-38fd98ff13c9.png)

**Comment by kezhenxu94** [comment, marked as answer]
You need to hit <kbd>Enter</kbd> after typing the keyword .

**Comment by wu-sheng** [comment]
This question existed in many issues and discussions already, please search before submit duplicated one.
